engine/.mega-linter.yml

```diff
@@ -0,0 +1,12 @@
+# Configuration file for MegaLinter
```

Comment:
We can actually have a single megalinter config and rely on flavors to control what gets run. That way, things like spell check and markdown rules are global, but dotnet linter only runs with dotnet flavor (because it's not included in the python flavor)